app/views/checklist_mailer/send_abandoned.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p><%= @checklist.name %> has been abandoned...</p>
```

Comment:
Think a bit more corporate here. Don't use the word 'abandoned'. Try:

You have an unused checklist open in Chekhov: "Checklist Name".

Please _link_here_ close it if you are no longer using it.

Thank you,
The Chekhov Robot

(something like that)